dapr/actor/runtime/runtime.py

```diff
@@ -118,21 +118,24 @@ async def fire_reminder(
         await manager.fire_reminder(ActorId(actor_id), name, request_body)
 
     @classmethod
-    async def fire_timer(cls, actor_type_name: str, actor_id: str, name: str) -> None:
+    async def fire_timer(
+            cls, actor_type_name: str, actor_id: str,
+            name: str, request_body: bytes) -> None:
```

Comment:
nit: request_body -> state 

dotnet-sdk and java-sdk uses `state` isntead of `request_body`